Makefile

```diff
@@ -257,8 +258,9 @@ test: test-env-info test-codeception
 
 .PHONY: test-codeception
 test-codeception:
-	@docker-compose exec php-fpm sh -c 'cd tests && composer install --prefer-dist --no-progress'
-	@docker-compose exec php-fpm tests/vendor/bin/codecept run --xml=results.xml --html
+	@docker-compose exec php-fpm bash -c 'cd tests && composer install --prefer-dist --no-progress'
+	@docker-compose exec php-fpm tests/vendor/bin/codecept run wpunit --xml=junit.xml --html --debug
+	@docker-compose exec php-fpm tests/vendor/bin/codecept run acceptance --xml=junit.xml --html
```

Comment:
Is it worth splitting this rule into three different targets so they can each be run independently? eg, `install-codeception test-codeception-unit test-codeception-acceptance` ?

Reply:
Yes, good suggestion :+1: will split it